Shared/Controllers/AppContext.swift

```diff
@@ -66,6 +70,18 @@ final class AppContext: ObservableObject {
         didSet { AudioManager.shared.isSpeakerOutputPreferred = preferSpeakerOutput }
     }
 
+    #if os(iOS)
+        // Krisp noise filter example
+        lazy var krispProcessor = LiveKitKrispNoiseFilter()
```

Comment:
@hiroshihorie can we add `ObservableObject` to `LiveKitKrispNoiseFilter` so it can be passed in `@EnvironmentObject`? That would be a more intuitive way to integrate into SwiftUI